hookman/hooks.py

```diff
@@ -129,3 +137,36 @@ def _bind_libs_functions_on_hook_caller(self, shared_lib_path: Path, hook_caller
         for hook in hooks_to_bind:
             cpp_func = getattr(hook_caller, hook)
             cpp_func(hooks_to_bind[hook])
+
+
+    def plugins_has_conflicts(self) -> bool:
+        """
+        Auxiliary methods that checks if the get_status has any conflict
+        """
+        if self.get_status():
+            return True
+        else:
+            return False
+
+    def get_status(self) -> List[Optional[ConflictStatus]]:
+        """
+        Check if the plugins has conflicts between then.
+        If a conflict is found a list of ConflictStatus object will be returned.
+        Otherwise a empty list is returned.
+        """
+        list_of_conflicts = []
+        plugins_available = self.plugins_available()
+        if not plugins_available:
+            return list_of_conflicts
+
+        hooks_status = {hook_name: [] for hook_name in plugins_available[0].hooks_available.keys()}
```

Comment:
Hmm two comments:

1. When `PluginInfo.hooks_available` dict is filled? Shouldn't this information come from `Hookman`?

2. You don't need to initialize `hooks_status` at all, you only need to know duplicated *implementations*. If you change this to `hooks_status = {}` this code should still work IIUC.

Reply:
1. PluginInfo is filled on `plugins_available` method and this method is part of Hookman, so technically this information it's coming from Hookman 😜 :
```Python
def plugins_available(self) -> Optional[List[PluginInfo]]:
        plugin_config_files = hookman_utils.find_config_files(self.plugins_dirs)
        return [PluginInfo(plugin_file, self.hooks_available) for plugin_file in plugin_config_files]
```

Reply:
2. `hooks_status ` is a dictionary with that holds the name of the hook, and a list with the plugins that implements this hook.
Ex. `hooks_status = {'env_temperature': [ Plugin1, Plugin2 ] }`

Currently, the only check that I'm doing it's if more than one plugin implements the same hook (as can be checked on the second loop).